### PR TITLE
4589 action klaviyo

### PR DESCRIPTION
**Pull request by luancazarine**
Actions:
  - Create New List
  - Add Members to a List
  - Get Lists (refactored)

<a href="https://gitpod.io/#https://github.com/PipedreamHQ/pipedream/pull/4700"></a>

### CODE REVIEW

**Comment by dylburger** [comment]
Thanks for submitting this PR! When we review PRs, we follow the [Pipedream component guidelines](https://pipedream.com/docs/components/guidelines/). If you're not familiar, here's a quick checklist:

- [ ] [Create components to address specific use cases whenever possible](https://pipedream.com/docs/components/guidelines/#component-scope)
- [ ] [Component `key`s should follow the format `app_name_slug-slugified-component-name`](https://pipedream.com/docs/components/guidelines/#component-key-pattern)
- [ ] Components should follow the [standard directory structure](https://pipedream.com/docs/components/guidelines/#folder-structure)
- [ ] [Prefer Node.js client libraries to REST APIs](https://pipedream.com/docs/components/guidelines/#using-apis-vs-client-libraries)
- [ ] When making API requests, [handle pagination](https://pipedream.com/docs/components/guidelines/#pagination) to ensure all data / events are processed
- [ ] [Use `secret` props](https://pipedream.com/docs/components/guidelines/#capturing-sensitive-data) to capture sensitive data
- [ ] Props and methods should be defined in [app files](https://pipedream.com/docs/components/guidelines/#app-files) whenever possible
- [ ] Document methods with [JS Docs](https://pipedream.com/docs/components/guidelines/#methods)
- [ ] [Use `optional` props](https://pipedream.com/docs/components/guidelines/#optional-vs-required-props) whenever possible, and [set a `default` value](https://pipedream.com/docs/components/guidelines/#default-values) where you can
- [ ] Use [async options](https://pipedream.com/docs/components/guidelines/#async-options) to accept user input wherever possible

File: components/klaviyo/actions/create-new-list/create-new-list.mjs [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import klaviyo from "../../klaviyo.app.mjs";
+
+export default {
+  key: "klaviyo-create-new-list",
+  name: "Create New List",
+  description: "Creates a new list in an account.",
```
Comment (andrewjschuang) [inline components/klaviyo/actions/create-new-list/create-new-list.mjs:6]: Can you add the link to the API docs here please?

File: components/klaviyo/actions/get-lists/get-lists.mjs [hunk 2]
```diff
@@ -1,24 +1,23 @@
-// legacy_hash_id: a_2wimkG
-import { axios } from "@pipedream/platform";
+import klaviyo from "../../klaviyo.app.mjs";
 
 export default {
   key: "klaviyo-get-lists",
   name: "Get Lists",
   description: "Get a listing of all of the lists in an account.",
```
Comment (andrewjschuang) [inline components/klaviyo/actions/get-lists/get-lists.mjs:6]: Can you add the link to the API docs here please?

File: components/klaviyo/actions/add-member-to-list/add-member-to-list.mjs [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+        country,
+        latitude,
+        longitude,
+        region,
+        zip,
+        timezone,
+      },
+    };
+
+    if (phone) opts.phone_number = phone;
```
Comment (andrewjschuang) [inline components/klaviyo/actions/add-member-to-list/add-member-to-list.mjs:185]: Can `phone_number` be declared in `opts` above?

File: components/klaviyo/actions/add-member-to-list/add-member-to-list.mjs [hunk 4]
```diff
@@ -0,0 +1,200 @@
+import klaviyo from "../../klaviyo.app.mjs";
+
+export default {
+  key: "klaviyo-add-member-to-list",
+  name: "Add Member To List",
+  description: "Add member to a specific list.",
```
Comment (andrewjschuang) [inline components/klaviyo/actions/add-member-to-list/add-member-to-list.mjs:6]: Can you add the link to the API docs here please?

**Review by andrewjschuang — CHANGES_REQUESTED** [review]
Thanks @luancazarine! Just a few minor comments!

File: components/klaviyo/actions/add-member-to-list/add-member-to-list.mjs (hunk 3 above)
Comment (luancazarine) [reply components/klaviyo/actions/add-member-to-list/add-member-to-list.mjs:185]: > Can `phone_number` be declared in `opts` above?

If i declare this field and it goes blank, the api returns 400 
image

**Comment by DilanAthukorala** [comment]
Verification completed, ready for release.

**Review by andrewjschuang — APPROVED** [review]

**Pull request merged**